### DISCUSSION TITLE
A question

### DISCUSSION THREAD

**Discussion by CelestineLee** [redraskal/r6-dissect#18] [Q&A] [answered]
Hey! I wonder if I can ask something. 
I learned that people can upload the csgo replay file and load it into blender, a modeling software. They can replay the gameplay in blender, with map file loaded too.
So I wonder if u can finally make this happen. This is so exciting if u can do it. 
I am familiar with blender and I used it to make videos of operators with action.
If there is anything I can help, plz tell me. I know little of coding. If u want to find a partner to assist u, plz let me know.
Hope the best for u.

**Comment by redraskal** [comment, marked as answer]
This sounds like a cool idea, but I doubt it is feasible with the complexity of siege. I hope to eventually retrieve movement data from replays. Maybe that can be used to help animate characters in Blender. I think the odds are greater for making a tool to modify replays and play them back inside siege. Modifying replays would not be possible for a long time if it becomes reality, but I would bet more on that workflow!

**Reply by CelestineLee** [reply]
I really wanna share with u this video. The author decode csgo replay file and rebuild the game play in blender. But this video's language is Chinese. The author taught all the steps he did. And he demonstrated the final result, which is so cool. I hope oneday we can make it.
And I hope this video might give u inspiration or somehow helpful for u.
The link is
https://www.bilibili.com/video/BV1Cf4y1g7Au/?spm_id_from=333.880.my_history.page.click&vd_source=77902ceb271e6fc4883e80dd27e51dda

**Reply by BadCoder1337** [reply]
Considering the colossal difference between CS: GO and R6 developer tools, this is completely impossible right now and in foreseeable future.
| | CS:GO | R6 |
| ------------- | ------------- | - |
| Map Editor | hammer | lol no |
| SDK | open source | wut? |
| Demos | since Doom | Y6S1 |
| Demo Compatibility | here are 10 versions of Overpass | let me delete it |
| Stats API/OAuth | fine | fuck off |
| Documentation | fine | we do not give it, plz do not reverse-engineer. |

**Reply by redraskal** [reply]
lol you killed me with that chart

**Reply by CelestineLee** [reply]
damn so professional
